item.py

```diff
@@ -0,0 +1,8 @@
+class Item0: 
+    def __init__(self, name, ability):
+        self.name = name
+        self.ability = ability
+
```

Comment:
You are missing methods! You want to be able to do something with or to your items. For example, you could examine them, you could do the actual ability, etc.